test/unit/client-side-encryption/client_encryption.test.ts

```diff
@@ -158,7 +201,7 @@ describe('ClientEncryption', function () {
 
         expect(createDataKeySpy.callCount).to.equal(0);
         const options = createCollectionSpy.getCall(0).args[1];
-        expect(options).to.deep.equal({ encryptedFields: { fields: 'not an array' } });
+        expect(options).to.containSubset({ encryptedFields: { fields: 'not an array' } });
```

Comment:
I assume this was changed to accommodate a new option? Was this the timeoutContext?

Reply:
Yeah, rather than containSubset I can also just add in timeoutContext